Makefile

```diff
@@ -6,11 +6,15 @@ RELEASE_NAME_SYSLOG := rsyslog
 # build configuration options
 SYSLOG_ENABLED  ?= false
 BEE2_ENABLED    ?= false
+DURATION_TIME   ?= 25s
+MINIO_ENABLED   ?= false
```

Comment:
Since the MinIO became our primary storage, it should be enabled by default, right?

Reply:
Not, sure it wasn't when minio functionality was added, so I think the author of these changes done this intentionally.